library/src/main/java/com/novoda/downloadmanager/DownloadMigratorBuilder.java

```diff
@@ -34,19 +48,28 @@ public static DownloadMigratorBuilder newInstance(Context context) {
                 customizer,
                 notificationChannelProvider
         );
-
         Handler handler = new Handler(Looper.getMainLooper());
-        return new DownloadMigratorBuilder(applicationContext, handler, notificationChannelProvider, defaultNotificationCreator);
+        MigrationCallback migrationCallback = (status) -> {
+            Log.v(status.toString());
```

Comment:
we might not need this for prod release

Reply:
Yea I was thinking we should do something similar for this as we did for `Merlin` https://github.com/novoda/merlin/blob/master/core/src/main/java/com/novoda/merlin/Logger.java

Where clients can specify a log handle and if they do then they receive debug messaging. Would be super useful if anyone ever opens an issue.